tests/__mocks__/handlers.ts

```diff
@@ -0,0 +1,11 @@
+import { http, HttpResponse } from "msw";
+import { db } from "./db";
+
+/**
+ * Intercepts the routes and returns a custom payload
+ */
+export const handlers = [
+  http.get("https://api.ubiquity.com/users", () => {
```

Comment:
Whats the URL?

Reply:
Just a placeholder, can be anything, only purpose is to provide an example

Reply:
yeah it's just a mock